src/notation/internal/notationuiactions.cpp

```diff
@@ -563,12 +563,18 @@ const UiActionList NotationUiActions::m_actions = {
              TranslatableString("action", "Measure properties…"),
              TranslatableString("action", "Measure properties…")
              ),
-    UiAction("add-remove-breaks",
+    UiAction("add-remove-breaks", // DEPRECATED
```

Comment:
I had left it in because one unit tests was still using this function and it appeared to be the only unit test covering line breaks. But at this point I'd better just remove that test and make a new one

Reply:
You removed those old tests but not yet the UiAction :)